src/Sentry.Unity.Editor/ConfigurationWindow/OptionsConfigurationTab.cs

```diff
@@ -118,8 +126,8 @@ internal static void SetScript(String scriptName)
             AssetDatabase.Refresh();
 
             // Don't overwrite already set OptionsConfiguration
-            var options = SentryWindow.Instance!.Options;
-            if (options.OptionsConfiguration == null)
+            var options = EditorWindow.GetWindow<SentryWindow>().Options;
```

Comment:
yes, I guess it didn't work since the wizard PR. `Instance` was always null on the next line because after the assemblies were reloaded, the window wasn't available in the static scope